core/templates/pages/admin-page/misc-tab/admin-misc-tab.directive.ts

```diff
@@ -155,6 +156,22 @@ angular.module('oppia').directive('adminMiscTab', [
             });
         };
 
+        ctrl.retrieveExplorationInteractionIds = function() {
```

Comment:
Can you add tests for this method?

Reply:
For this we'll be needing front-end test ? 🤔

Reply:
@DubeySandeep we need frontend test for this right?

Reply:
Yes.. @marianazangrossi can help you out if necessary!

Reply:
@BenHenning actually we can't any test for directives as of now @marianazangrossi will be doing this by breaking down things into components in her GSoC period

Reply:
I'll be doing this @aggarwalpulkit596, however, we encourage you to make the unit tests still. I'll be migrating both the directive file as its spec file as well.